### PR TITLE
some small fixes.

### PR DESCRIPTION
**Pull request by TingluoHuang**

### CODE REVIEW

File: src/Misc/layoutbin/en-US/strings.json [hunk 1]
```diff
@@ -185,11 +185,11 @@
     "bin/Agent.Listener configure --url http://mylocaltfsserver:8080/tfs --agent myagent --pool default --nostart --auth integrated..."
   ],
   "ListenForJobs": "{0:u}: Listening for Jobs",
-  "LocalServiceNotFound": "Can not find local service account",
   "MemberDoesNotExists": "Member: {0} does not Exist",
   "MissingAgent": "The agent no longer exists on the server. Please reconfigure the agent.",
   "MissingConfig": "Cannot connect to server, because config files are missing. Skipping removing agent from the server.",
   "N": "N",
+  "NetworkServiceNotFound": "Can not find network service account",
```
Comment (bryanmacfarlane) [inline src/Misc/layoutbin/en-US/strings.json]: cannot is one word

File: src/Agent.Listener/Configuration/NativeWindowsServiceHelper.cs [hunk 2]
```diff
@@ -324,15 +324,14 @@ public bool IsValidCredential(string domain, string userName, string logonPasswo
 
         public NTAccount GetDefaultServiceAccount()
         {
-            SecurityIdentifier sid = new SecurityIdentifier(WellKnownSidType.LocalServiceSid, domainSid: null);
+            SecurityIdentifier sid = new SecurityIdentifier(WellKnownSidType.NetworkServiceSid, domainSid: null);
```
Comment (ericsciple) [inline src/Agent.Listener/Configuration/NativeWindowsServiceHelper.cs]: Is it OK to always use NetworkService as the default? I would think it is fine, but I don't know why the old agent preferred LocalService on a non-domain joined machine.

Comment (TingluoHuang) [inline src/Agent.Listener/Configuration/NativeWindowsServiceHelper.cs]: LocalService/NetworkService/LocalSystem has nothing to do with domain joined or not. :)

File: src/Agent.Listener/MessageListener.cs [hunk 3]
```diff
@@ -270,12 +270,12 @@ private bool IsGetNextMessageExceptionRetriable(Exception ex)
                 ex is AccessDeniedException ||
                 ex is VssUnauthorizedException)
             {
-                Trace.Info($"Non-retriable exception: {ex.Message}");
+                Trace.Info($"Non-retryable exception: {ex.Message}");
```
Comment (ericsciple) [inline src/Agent.Listener/MessageListener.cs]: From a hasty web search, I can point you to three dictionary references for the word "retriable":
 * https://en.wiktionary.org/wiki/retriable
 * http://www.yourdictionary.com/retriable
 * http://www.wordow.com/english/dictionary/?t=retriable

I didn't find any dictionary entries for "retryable" (Urban Dictionary doesn't count).

**Comment by ericsciple** [comment]
looks good. couple small comments

**Pull request merged**